XLS-33d/README.md

```diff
@@ -0,0 +1,594 @@
+5. **No Rippling**. Unlike some existing capabilities of the ledger, CFTs are not eligible for [rippling](https://xrpl.org/rippling.html#rippling), and thus do not have any configurability settings related to that functionality.
+
+### 1.1.3 Release Timeline and Scope
+
+XLS-33 will only cover the addition of the new data structures for CFTs, integration with the **`Payment`** transaction such that users can make CFT to CFT payments (IE no cross currency payments), and several other incidental additions like new requirements for account deletion (discussed below). Later amendments will integrate CFTs with other features of the XRPL such as the DEX.
+
+## 1.2. Creating Compact Fungible Tokens
+
+### 1.2.1. On-Ledger Data Structures
+
+We propose two new objects and one new ledger structure:
+
+1. A **`CFTokenIssuance`** is a new object that describes a fungible token issuance created by an issuer.
+1. A **`CFToken`** is a new object that describes a single account's holdings of an issued token.
+1. A **`CFTokenPage`** is a ledger structure that contains a set of **`CFToken`** objects owned by the same token holder.
+
+#### 1.2.1.1. The **`CFTokenIssuance`** object
+
+The **`CFTokenIssuance`** object represents a single CFT issuance and holds data associated with the issuance itself. Token issuances are created using the **`CFTokenIssuanceCreate`** transaction and can, optionally, be destroyed by the **`CFTokenIssuanceDestroy`** transaction.
+
+##### 1.2.1.1.1. **`CFTokenIssuance`** Ledger Identifier
+The ID of an CFTokenIssuance object, a.k.a `CFTokenIssuanceID` is the result of SHA512-Half of the following values, concatenated in order:
+
+* The CFTokenIssuance space key (0x007E).
+* The AccountID of the issuer.
+* The AssetCode of the issuance.
+
+##### 1.2.1.1.2. Fields
+
+**`CFTokenIssuance`** objects are stored in the ledger and tracked in an [Owner Directory](https://xrpl.org/directorynode.html) owned by the issuer. Issuances have the following required and optional fields:
+
+| Field Name          | Required?          | JSON Type | Internal Type |
+| ------------------- |--------------------|-----------|---------------|
+| `LedgerEntryType`   | :heavy_check_mark: | `number`  | `UINT16`      |
+| `Flags`             | :heavy_check_mark: | `number`  | `UINT32`      |
+| `Issuer`            | :heavy_check_mark: | `string`  | `ACCOUNTID`   |
+| `AssetCode`         | :heavy_check_mark: | `string`  | `UINT160`     |
+| `AssetScale`        | (default)          | `number`  | `UINT8`       |
+| `MaximumAmount`     | :heavy_check_mark: | `string`  | `UINT64`      |
+| `OutstandingAmount` | :heavy_check_mark: | `string`  | `UINT64`      |
+| `LockedAmount`      | ️(default)          | `string`  | `UINT64`      |
+| `TransferFee`       | ️(default)          | `number`  | `UINT16`      |
+| `Metadata`          |                    | `string`  | `BLOB`        |
```

Comment:
```suggestion
| `CFTMetadata`          |                    | `string`  | `BLOB`        |
```

I think we'll need to change this name so it doesn't collide with `sfMetadata` which is a `METADATA` type that we can't (or don't want to) use for CFT (in the rippled codebase, a `BLOB` is actually an `STI_VL` type).